Add first draft of terms and conditions and about (#38)
huserben/scrum-kata: terms_conditions.md

---
layout: default
title: Terms & Conditions
parent: Contribute
---

# Terms and conditions for Contributions

Thanks for thinking about contributing. Our aim is to involve the community as much as possible.

Before you contribute a question, please be aware about the terms and conditions that apply for questions:

1. By submitting content, you give away any rights you may have on it. No legal claims can be made on material submitted. Your submitted question will be hosted on GitHub under an [MIT License](https://github.com/huserben/scrum-kata/blob/main/LICENSE).
2. Therefore, only material you yourself have created and own may be submitted.
3. We reserve the right to make changes to your question or to not include it at all.
4. Submitting third-party material is not allowed, unless you've double checked with the original authors. Please reach out to us if you have questions.

> 5. If you suspect your own or third-party material is being used on this site, please send a message immediately. The offending material will be removed as soon as possible.

Reviewer: ```suggestion
5. If you suspect your own or third-party material is being used on this site, please [send a message](/scrum-kata/about/#contact) immediately. The offending material will be removed as soon as possible.
```

just for users' convenience :)

6. Contributions can be made via GitHub and need a GitHub account. Some terms and conditions from GitHub may apply.

> 7. Contributions will be visible on GitHub and on the contributions page. If questions are submitted via GitHub, this is automatically included in the contributions section. If you submit questions otherwise we'll open to discuss how to credit you.

Reviewer: ```suggestion
7. Contributions will be visible on GitHub and the contributions page. If questions are submitted via GitHub, this will automatically include authors in the contributions section. If you submit questions otherwise, we are open to discussing how to credit you.
```
